### PR TITLE
Update current version with latest inner source version

### PR DESCRIPTION
**Pull request by bruno-brant**
This commits adds a bunch of corrections and updates that were done on a _inner source_ branch and was still missing from the public repo.

### CODE REVIEW

File: src/Liquid.OnAzure/Storages/AzureBlob.cs [hunk 1]
```diff
@@ -96,21 +122,20 @@ private byte[] ReadFully(Stream input, int size)
             }
         }
 
-        public async void InsertUpdateAsync(ILightAttachment attachment)
+        public async Task InsertUpdateAsync(ILightAttachment attachment)
```
Comment (lucianareginalino) [inline src/Liquid.OnAzure/Storages/AzureBlob.cs:99]: This method should return an ILightAttachment to comply with the inner source version.

File: src/Liquid.Base/Interfaces/Repository/ILightMediaStorage.cs [hunk 2]
```diff
@@ -9,10 +9,10 @@ namespace Liquid.Interfaces
...
     public interface ILightMediaStorage : IWorkBenchHealthCheck
     {	
-		String Conection { get; set; }
-        String Container { get; set; }
+		string Conection { get; set; }
+        string Container { get; set; }
         Task<ILightAttachment> GetAsync(string resourceId, string id);
-        void InsertUpdateAsync(ILightAttachment attachment); 
-        void Remove(ILightAttachment attachment); 
+        Task InsertUpdateAsync(ILightAttachment attachment);
```
Comment (lucianareginalino) [inline src/Liquid.Base/Interfaces/Repository/ILightMediaStorage.cs:15]: This method should return an ILightAttachment to comply with the inner source version.

**Review by lucianareginalino — APPROVED** [review]

File: src/Liquid.OnAzure/Storages/AzureBlob.cs (hunk 1 above)
Comment (bruno-brant) [reply src/Liquid.OnAzure/Storages/AzureBlob.cs:99]: The changes that SAS required are being moved to a different branch. We must discuss how to better approach it.

File: src/Liquid.Base/Interfaces/Repository/ILightMediaStorage.cs (hunk 2 above)
Comment (bruno-brant) [reply src/Liquid.Base/Interfaces/Repository/ILightMediaStorage.cs:15]: SAS was moved to another PR, and we must rethink that approach.

**Pull request merged**